datadog/tests/cassettes/TestAccDatadogLogsArchiveAzure_basic.yaml

```diff
@@ -64,11 +64,11 @@ interactions:
     headers:
       Accept:
       - application/json
-    url: https://api.datadoghq.com/api/v2/logs/config/archives/-GWdZ6m2RhKwHu40SqvtKg
+    url: https://api.datadoghq.com/api/v2/logs/config/archives/j0FZmzs-QjusCqvWcPSSsQ
```

Comment:
What are these urls for? Why are we testing them here?

Reply:
its an integration test that tests that we can create an azure integration, and then create a log archive connected to the same tenant id. since the azure integration was changed, this test was re-recorded since it uses the azure integration.